crates/biome_css_parser/src/syntax/selector/attribute.rs

```diff
@@ -0,0 +1,94 @@
+use crate::parser::CssParser;
+use crate::syntax::parse_error::{
+    expect_any_attribute_matcher_name, expect_any_attribute_modifier, expected_identifier,
+};
+use crate::syntax::selector::selector_lex_context;
+use crate::syntax::{is_at_identifier, parse_css_string, parse_regular_identifier};
+use biome_css_syntax::CssSyntaxKind::*;
+use biome_css_syntax::{CssSyntaxKind, T};
+use biome_parser::diagnostic::expected_token;
+use biome_parser::parse_recovery::ParseRecovery;
+use biome_parser::prelude::ParsedSyntax;
+use biome_parser::prelude::ParsedSyntax::{Absent, Present};
+use biome_parser::{token_set, Parser, TokenSet};
+
+const ATTRIBUTE_SELECTOR_RECOVERY_SET: TokenSet<CssSyntaxKind> = token_set![T![')'], T!['{']];
```

Comment:
Can you add a small comment that shows why/when these tokens do a recovery? I can guess two cases:

```css
:hover) {
/*    ^ recovers here */
:hover {
/*     ^ recovers here */
```

Reply:
Sure,
I was wondering if we could do it in another PR? 
I'd like to merge this one and the next with pseudo class nth selector.

After I'm going to add error cases for every pseudo class selector.
What do you think?

Reply:
Sounds good 😊